docusaurus/docs/iOS/basics/integration.md

```diff
@@ -0,0 +1,184 @@
+  - `pod 'StreamChat-XCFramework', '~> 4.0.0'`
+- For the UIKit components (**StreamChatUI**, which depends on **StreamChat**) use:
+  - `pod 'StreamChatUI-XCFramework', '~> 4.0.0'`
+
+</p>
+</details>
+
+Example: If you want to use the UIKit components, it should look similar the snippet below.
+
+```ruby
+target 'MyProject' do
+  use_frameworks!
+
+  # Pods for MyProject
+  pod 'StreamChatUI', '~> 4.0.0'
+end
+```
+
+StreamChatUI and StreamChatSwiftUI pod will automatically include the StreamChat dependency. If you want just the StreamChat dependency, without the UI components, use StreamChat as shown above.
+
+Now that we’ve modified our Podfile, let’s go ahead and install the project dependencies via the terminal with one simple command:
+
+```bash
+pod install --repo-update
+```
+
+The above command will generate the **MyProject.xcworkspace** file automatically.
+
+With our workspace now containing our Pods project with dependencies, as well as our original project, let’s go ahead and move over to Xcode to complete the process.
+
+_More information about CocoaPods [can be found here](https://cocoapods.org/)._
+
+### Carthage
+
+In your project's `Cartfile`, add one of these options
+
+- For the LLC (**StreamChat**) use:
+  - `github "getstream/stream-chat-swift" ~> 4.6.0`
+- For the UIKit components (**StreamChatUI**, which depends on **StreamChat**) use:
+  - `github "getstream/stream-chat-swift" ~> 4.0.0`
+
+:::note
+Our SwiftUI components library is not yet available using Carthage, please use Swift Package Manager or CocoaPods.
+:::
+
+
+<details><summary>➤ Do you want to use <b>XCFrameworks</b>? (Click to read more)</summary>
+<p>
+
+:::caution
+Our XCFrameworks are built with **Swift 5.5**. In order to use them you need **Xcode 13** or above
+:::
+
+You can learn more about [our Module Stable XCFrameworks here](#xcframeworks)
+
+- For the LLC (**StreamChat**) use:
+  - `binary "https://raw.githubusercontent.com/GetStream/stream-chat-swift/main/StreamChatArtifacts.json" ~> 4.6`
+- For the UIKit components (**StreamChatUI**, which depends on **StreamChat**) use:
+  - `binary "https://raw.githubusercontent.com/GetStream/stream-chat-swift/main/StreamChatArtifacts.json" ~> 4.0`
+
+</p>
+</details>
+
+Now that we’ve modified our Cartfile, let’s go ahead and install the project dependencies via the terminal with one simple command:
+
+```bash
+carthage update --use-xcframeworks
+```
+
+This command will create pre-built XCFrameworks built from our source code. You now need to add those to your project.
+
+Open the `Carthage/Build` folder that has been created in the root of your project, and drag and drop the frameworks you want to use. Those should be added to the "Frameworks, Libraries, and Embedded Content" section under General settings:
+
+![Screenshot shows XCFrameworks being draged into Xcode](../assets/carthage-drag.png)
+
+_More information about Carthage [can be found here](https://github.com/Carthage/Carthage)._
```

Comment:
It's good practice to explicitly call to `Embedd & Sign` the xcframeworks as the last integration step.

Reply:
Good point